source/stylesheets/header.scss

```diff
@@ -22,6 +24,8 @@
   &::-webkit-details-marker {
     display: none;
   }
+  
+  &::-moz-list-bullet { list-style-type: none; }
```

Comment:
Can you break the declaration into multiple lines like the others, for consistency?